### PR TITLE
Added player movement animations

### PR DESCRIPTION
**Pull request by TJ-Smits**
ADDED PLAYER ANIMATIONS FOR WALKING AND STANDING IDLE. IT LOOKS GOOD

### CODE REVIEW

File: behind-the-tankard/Assets/Scripts/Interactable/PlayerMovement.cs [hunk 1]
```diff
@@ -60,6 +62,11 @@ void Update()
         PlayerInput();
 
         ApplyGravity();
+
+        if (_animator.GetBool("isWalking") != isWalking)
+        {
+            _animator.SetBool("isWalking", isWalking);
```
Comment (YWAMWeessies) [inline behind-the-tankard/Assets/Scripts/Interactable/PlayerMovement.cs]: change to trigger

File: behind-the-tankard/Assets/Scripts/Interactable/PlayerMovement.cs [hunk 2]
```diff
@@ -39,13 +38,16 @@ public class PlayerMovement : MonoBehaviour, IInventorist
     public Transform inventoryTransform;
     private Inventory _inventory;
 
+    private Animator _animator;
+    private bool isWalking = false;
```
Comment (YWAMWeessies) [inline behind-the-tankard/Assets/Scripts/Interactable/PlayerMovement.cs]: remove unused variable

Comment (TJ-Smits) [reply behind-the-tankard/Assets/Scripts/Interactable/PlayerMovement.cs]: done for

File: behind-the-tankard/Assets/Scripts/Interactable/PlayerMovement.cs (hunk 1 above)
Comment (TJ-Smits) [reply behind-the-tankard/Assets/Scripts/Interactable/PlayerMovement.cs]: Changed to trigger

**Pull request merged**